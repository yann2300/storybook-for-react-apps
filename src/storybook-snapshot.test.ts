import initStoryshots from '@storybook/addon-storyshots'

// This will automate snapshot testing for all stories
initStoryshots({
  storyKindRegex: /^Components/,
  storyNameRegex: 'Default',
})
