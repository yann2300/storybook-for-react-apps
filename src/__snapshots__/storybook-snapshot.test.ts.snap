// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Components/Badge Default 1`] = `
<div
  className="sc-iqseJM eiMYDz"
>
  <span
    className="sc-bdvvtL eBjqRR"
    fontWeight="regular"
    size="S"
  >
    Hamburgers
  </span>
</div>
`;

exports[`Storyshots Components/Button Default 1`] = `
<button
  className="sc-dlVxhl kMLmuM"
  type="button"
>
  Button
</button>
`;

exports[`Storyshots Components/Category Default 1`] = `
<figure
  className="sc-eCImPb bFUXaa"
  data-testid="Pizza"
>
  <img
    alt="restaurant category"
    className="sc-jRQBWg bKOraV"
    src="https://images.pexels.com/photos/2147491/pexels-photo-2147491.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=550"
  />
  <figcaption
    className="sc-iCfMLu eVTlNg"
  >
    <span
      className="sc-bdvvtL iohvTk"
      fontWeight="medium"
      size=""
    >
      Pizza
    </span>
  </figcaption>
</figure>
`;

exports[`Storyshots Components/ErrorBlock Default 1`] = `
<div
  className="sc-kfPuZi cVOxMm"
>
  <h2
    className="sc-gsDKAQ gXMQwD"
  >
    This is not the food youâ€™re looking for.
  </h2>
  <div
    className="sc-fKVqWL ccLwLP"
  >
    <img
      alt="no restaurants found"
      src="sushi.svg"
    />
  </div>
  <p
    className="sc-bdvvtL eMIAGF"
    fontWeight="regular"
    size=""
  >
    There seems that there are no restaurants in this category yet. Try to come back later?
  </p>
  <button
    className="sc-dlVxhl kMLmuM"
    type="button"
  >
    See all restaurants
  </button>
</div>
`;

exports[`Storyshots Components/Footer Default 1`] = `
<div
  style={
    Object {
      "height": "100vh",
      "position": "relative",
    }
  }
>
  <div
    style={
      Object {
        "bottom": 0,
        "position": "absolute",
        "width": "100%",
      }
    }
  >
    <div
      className="sc-fXEqDS gtxefw"
    >
      <div
        className="container"
      >
        <div
          className="sc-FNXRL crHivn"
        >
          <div
            className="sc-bilyIR bOZwfU"
          >
            <svg
              className="sc-uojGG dLYBRT"
              fill="none"
              viewBox="0 0 23 20"
            >
              <path
                className="logo--face-left"
                d="M11.74 19.1662L6.66566 14.0788L0.82428 11.0619L1.76834 4.49574L0.82428 0.029541L11.74 2.1887V19.1662Z"
                fill="#61D8DE"
              />
              <path
                className="logo--face-right"
                d="M11.74 19.1366L16.8143 14.0493L22.6557 11.0619L21.7116 4.49574L22.6557 0.029541L11.74 2.1887V19.1366Z"
                fill="#4CC8C0"
              />
              <path
                className="logo--ear-bottom-right"
                d="M22.6262 10.8549L14.4246 2.72113L22.6262 0L21.7411 4.61408L22.6262 10.8549Z"
                fill="#36C1BF"
              />
              <path
                className="logo--ear-bottom-left"
                d="M0.883118 10.8549L9.11415 2.69155L0.883118 0L1.79768 4.61408L0.883118 10.8549Z"
                fill="#36C1BF"
              />
              <path
                className="logo--ear-top-right"
                d="M21.7115 4.55497C21.564 4.58455 14.454 2.78033 14.454 2.78033L22.5671 0.0592041L21.7115 4.55497Z"
                fill="#22ACA7"
              />
              <path
                className="logo--ear-top-left"
                d="M1.79794 4.58452C1.94545 4.6141 9.05541 2.80987 9.05541 2.80987L0.942383 0.0887451L1.79794 4.58452Z"
                fill="#22ACA7"
              />
              <path
                className="logo--nose"
                d="M13.3626 17.5394L11.74 16.4451L10.1174 17.5394L11.74 19.1366L13.3626 17.5394Z"
                fill="#B1DCE3"
              />
            </svg>
          </div>
          <div
            className="sc-bQtKYq eEghbS"
          >
            <h2
              className="sc-gsDKAQ gXMQwD"
            >
              Discover us
            </h2>
            <ul>
              <li>
                <p
                  className="sc-bdvvtL eMIAGF"
                  fontWeight="regular"
                  size=""
                >
                  <a
                    href="/"
                    onClick={[Function]}
                  >
                    Home
                  </a>
                </p>
              </li>
              <li>
                <p
                  className="sc-bdvvtL eMIAGF"
                  fontWeight="regular"
                  size=""
                >
                  <a
                    href="/categories"
                    onClick={[Function]}
                  >
                    Categories
                  </a>
                </p>
              </li>
              <li>
                <p
                  className="sc-bdvvtL eMIAGF"
                  fontWeight="regular"
                  size=""
                >
                  <a
                    href="/about"
                    onClick={[Function]}
                  >
                    About
                  </a>
                </p>
              </li>
              <li>
                <p
                  className="sc-bdvvtL eMIAGF"
                  fontWeight="regular"
                  size=""
                >
                  <a
                    href="/login"
                    onClick={[Function]}
                  >
                    Login
                  </a>
                </p>
              </li>
            </ul>
          </div>
          <div
            className="sc-bQtKYq eEghbS"
          >
            <h2
              className="sc-gsDKAQ gXMQwD"
            >
              Our social media
            </h2>
            <ul>
              <li>
                <p
                  className="sc-bdvvtL eMIAGF"
                  fontWeight="regular"
                  size=""
                >
                  <a
                    href="https://facebook.com"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    Facebook
                  </a>
                </p>
              </li>
              <li>
                <p
                  className="sc-bdvvtL eMIAGF"
                  fontWeight="regular"
                  size=""
                >
                  <a
                    href="https://instagram.com"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    Instagram
                  </a>
                </p>
              </li>
              <li>
                <p
                  className="sc-bdvvtL eMIAGF"
                  fontWeight="regular"
                  size=""
                >
                  <a
                    href="https://twitter.com"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    Twitter
                  </a>
                </p>
              </li>
            </ul>
          </div>
          <div
            className="sc-bQtKYq eEghbS"
          >
            <h2
              className="sc-gsDKAQ gXMQwD"
            >
              Check our apps
            </h2>
            <div
              className="footer-bottom"
            >
              <img
                alt="app store link"
                src="appstore-banner.png"
                style={
                  Object {
                    "marginBottom": "0.5rem",
                    "width": "120px",
                  }
                }
              />
              <img
                alt="google play link"
                src="googleplay-banner.png"
                style={
                  Object {
                    "width": "120px",
                  }
                }
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Components/Form/Input Default 1`] = `
<div
  className="sc-jOxtWs dPmJiL"
>
  <input
    aria-label="input"
    type="text"
  />
  
</div>
`;

exports[`Storyshots Components/Form/Select Default 1`] = `
<div
  className="sc-dFtzxp eiOnIo"
>
  <select
    aria-label="food"
    id="select"
    onChange={[Function]}
    value=""
  >
    <option
      value="Burger"
    >
      Burger
    </option>
    <option
      value="Pizza"
    >
      Pizza
    </option>
    <option
      value="Sushi"
    >
      Sushi
    </option>
  </select>
  
</div>
`;

exports[`Storyshots Components/Header Default 1`] = `
<div
  style={
    Object {
      "height": "100vh",
      "position": "relative",
    }
  }
>
  <div
    className="sc-hAcGzb kzLetd"
    data-testid="header"
  >
    <a
      aria-label="go to home page"
      className="sc-kYHfwS laOJvu"
      href="/"
      onClick={[Function]}
    >
      <div
        className="sc-bilyIR bOZwfU"
      >
        <svg
          className="sc-uojGG hNUAPL"
          fill="none"
          viewBox="0 0 23 20"
        >
          <path
            className="logo--face-left"
            d="M11.74 19.1662L6.66566 14.0788L0.82428 11.0619L1.76834 4.49574L0.82428 0.029541L11.74 2.1887V19.1662Z"
            fill="#61D8DE"
          />
          <path
            className="logo--face-right"
            d="M11.74 19.1366L16.8143 14.0493L22.6557 11.0619L21.7116 4.49574L22.6557 0.029541L11.74 2.1887V19.1366Z"
            fill="#4CC8C0"
          />
          <path
            className="logo--ear-bottom-right"
            d="M22.6262 10.8549L14.4246 2.72113L22.6262 0L21.7411 4.61408L22.6262 10.8549Z"
            fill="#36C1BF"
          />
          <path
            className="logo--ear-bottom-left"
            d="M0.883118 10.8549L9.11415 2.69155L0.883118 0L1.79768 4.61408L0.883118 10.8549Z"
            fill="#36C1BF"
          />
          <path
            className="logo--ear-top-right"
            d="M21.7115 4.55497C21.564 4.58455 14.454 2.78033 14.454 2.78033L22.5671 0.0592041L21.7115 4.55497Z"
            fill="#22ACA7"
          />
          <path
            className="logo--ear-top-left"
            d="M1.79794 4.58452C1.94545 4.6141 9.05541 2.80987 9.05541 2.80987L0.942383 0.0887451L1.79794 4.58452Z"
            fill="#22ACA7"
          />
          <path
            className="logo--nose"
            d="M13.3626 17.5394L11.74 16.4451L10.1174 17.5394L11.74 19.1366L13.3626 17.5394Z"
            fill="#B1DCE3"
          />
        </svg>
        <h4
          className="sc-gsDKAQ dWWSwS sc-eGPXGI kwRYLn"
        >
          MealDrop
        </h4>
      </div>
    </a>
    <div
      className="sc-xiLah caetdy"
    >
      <span
        className="navigation-items"
      >
        <button
          aria-label="turn on dark mode"
          className="sc-dlVxhl hCHqCI"
          onClick={[Function]}
          type="button"
        >
          <svg
            className="sc-dJjYzT eOfYGC"
            height="1.5rem"
            stroke="#2C2C2C "
            style={
              Object {
                "minWidth": "1.5rem",
              }
            }
            width="1.5rem"
          >
            <use
              xlinkHref="sprite-map.svg#sun"
            />
          </svg>
        </button>
        <a
          href="/"
          onClick={[Function]}
          tabIndex={-1}
        >
          <button
            className="sc-dlVxhl jWTTFi"
            type="button"
          >
            Home
          </button>
        </a>
        <a
          href="/categories"
          onClick={[Function]}
          tabIndex={-1}
        >
          <button
            className="sc-dlVxhl jWTTFi"
            type="button"
          >
            All restaurants
          </button>
        </a>
      </span>
      <button
        aria-label="food cart"
        className="sc-dlVxhl bYasiv"
        onClick={[Function]}
        type="button"
      >
        <svg
          className="sc-dJjYzT eOfYGC"
          height="1.5rem"
          stroke="#FFFFFF "
          style={
            Object {
              "minWidth": "1.5rem",
            }
          }
          width="1.5rem"
        >
          <use
            xlinkHref="sprite-map.svg#cart"
          />
        </svg>
      </button>
    </div>
  </div>
</div>
`;

exports[`Storyshots Components/IconButton Default 1`] = `
<button
  aria-label="forward"
  className="sc-cTAqQK EdzIK"
  type="button"
>
  <svg
    className="sc-dJjYzT eOfYGC"
    height={24}
    stroke="#202020"
    style={
      Object {
        "minWidth": 24,
      }
    }
    width={24}
  >
    <use
      xlinkHref="sprite-map.svg#arrow-right"
    />
  </svg>
</button>
`;

exports[`Storyshots Components/OrderSummary Default 1`] = `
<div
  className="sc-dvQaRk gOBXWZ"
>
  <h2
    className="sc-gsDKAQ gXMQwD sc-fWCJzd dWueft"
  >
    Your order
  </h2>
  <div
    className="sc-ZOtfp ekduDK"
  >
    <div
      className="sc-gGCDDS iPJmom"
    >
      <span
        className="sc-bdvvtL eMIAGF sc-clIzBv jdwou"
        fontWeight="regular"
        size=""
      >
        1
      </span>
      <span
        className="sc-bdvvtL eMIAGF sc-faUpoM ipGkSt"
        fontWeight="regular"
        size=""
      >
        Fries
      </span>
      <span
        className="sc-bdvvtL eMIAGF sc-Galmp jNhOJl"
        fontWeight="regular"
        size=""
      >
        â‚¬2.50
      </span>
    </div>
    <div
      className="sc-gGCDDS iPJmom"
    >
      <span
        className="sc-bdvvtL eMIAGF sc-clIzBv jdwou"
        fontWeight="regular"
        size=""
      >
        1
      </span>
      <span
        className="sc-bdvvtL eMIAGF sc-faUpoM ipGkSt"
        fontWeight="regular"
        size=""
      >
        Cheeseburger
      </span>
      <span
        className="sc-bdvvtL eMIAGF sc-Galmp jNhOJl"
        fontWeight="regular"
        size=""
      >
        â‚¬8.50
      </span>
    </div>
    <div
      className="sc-gGCDDS iPJmom"
    >
      <span
        className="sc-bdvvtL eMIAGF sc-clIzBv jdwou"
        fontWeight="regular"
        size=""
      >
        1
      </span>
      <span
        className="sc-bdvvtL eMIAGF sc-faUpoM ipGkSt"
        fontWeight="regular"
        size=""
      >
        Vanilla ice cream
      </span>
      <span
        className="sc-bdvvtL eMIAGF sc-Galmp jNhOJl"
        fontWeight="regular"
        size=""
      >
        â‚¬2.00
      </span>
    </div>
    <div
      className="sc-gGCDDS iPJmom"
    >
      <span
        className="sc-bdvvtL eMIAGF sc-clIzBv jdwou"
        fontWeight="regular"
        size=""
      >
        1
      </span>
      <span
        className="sc-bdvvtL eMIAGF sc-faUpoM ipGkSt"
        fontWeight="regular"
        size=""
      >
        Coca-Cola
      </span>
      <span
        className="sc-bdvvtL eMIAGF sc-Galmp jNhOJl"
        fontWeight="regular"
        size=""
      >
        â‚¬1.75
      </span>
    </div>
    <div
      className="sc-gGCDDS iPJmom"
    >
      <span
        className="sc-bdvvtL eMIAGF sc-clIzBv jdwou"
        fontWeight="regular"
        size=""
      >
        1
      </span>
      <span
        className="sc-bdvvtL eMIAGF sc-faUpoM ipGkSt"
        fontWeight="regular"
        size=""
      >
        Fanta
      </span>
      <span
        className="sc-bdvvtL eMIAGF sc-Galmp jNhOJl"
        fontWeight="regular"
        size=""
      >
        â‚¬1.50
      </span>
    </div>
    <div
      className="sc-gGCDDS iPJmom"
    >
      <span
        className="sc-bdvvtL eMIAGF sc-clIzBv jdwou"
        fontWeight="regular"
        size=""
      >
        1
      </span>
      <span
        className="sc-bdvvtL eMIAGF sc-faUpoM ipGkSt"
        fontWeight="regular"
        size=""
      >
        Sprite
      </span>
      <span
        className="sc-bdvvtL eMIAGF sc-Galmp jNhOJl"
        fontWeight="regular"
        size=""
      >
        â‚¬1.50
      </span>
    </div>
  </div>
  <div
    className="sc-TBWPX jBuQFi"
  >
    <p
      className="sc-bdvvtL eMIAGF"
      fontWeight="regular"
      size=""
    >
      Total
    </p>
    <h2
      className="sc-gsDKAQ gXMQwD sc-fWCJzd fhEUXd"
    >
      â‚¬17.75
    </h2>
  </div>
</div>
`;

exports[`Storyshots Components/PageSection Default 1`] = `
<div
  className="sc-jObWnj jjuOBc container-desktop"
>
  <div
    className="sc-dPiLbb dpblKd"
  >
    <h2
      className="sc-gsDKAQ gXMQwD"
    >
      Asian
    </h2>
  </div>
  <h1>
    Hello Dummy Content
  </h1>
</div>
`;

exports[`Storyshots Components/RestaurantCard Default 1`] = `
<div
  className="sc-egiyK icJZtF"
  data-testid="restaurant-card"
>
  <div
    style={
      Object {
        "display": "flex",
        "position": "relative",
      }
    }
  >
    <img
      alt="restaurant"
      className="sc-fotOHu jLcCjg"
      loading="lazy"
      src="https://images.unsplash.com/photo-1572802419224-296b0aeee0d9?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1003&q=20"
    />
  </div>
  <div
    className="sc-bqiRlB irQDHE"
  >
    <h2
      className="sc-gsDKAQ gXMQwD sc-ieecCq hThQmk"
    >
      Burger King
       
    </h2>
    <div
      className="sc-crHmcD llfpGV"
    >
      <span
        className="sc-bdvvtL eBjqRR review-text"
        fontWeight="regular"
        size="S"
      >
        â˜… 4.2 Very good
      </span>
    </div>
    <p
      className="sc-bdvvtL eMIAGF sc-fFeiMQ czcCJA"
      fontWeight="regular"
      size=""
    >
      Nicest place for burgers
    </p>
    <div
      className="sc-iqseJM eiMYDz sc-bkkeKt IJYNz"
    >
      <span
        className="sc-bdvvtL eBjqRR"
        fontWeight="regular"
        size="S"
      >
        comfort-food
      </span>
    </div>
  </div>
</div>
`;

exports[`Storyshots Components/Review Default 1`] = `
<div
  className="sc-crHmcD llfpGV"
>
  <span
    className="sc-bdvvtL eBjqRR review-text"
    fontWeight="regular"
    size="S"
  >
    No reviews yet
  </span>
</div>
`;

exports[`Storyshots Components/Spinner Default 1`] = `
<div
  style={
    Object {
      "minHeight": "100vh",
    }
  }
>
  <svg
    className="sc-cxpSdN fnhyRf"
    preserveAspectRatio="xMidYMid"
    viewBox="0 0 100 100"
    xmlns="http://www.w3.org/2000/svg"
  >
    <text
      fill="#2C2C2C "
      transform="translate(-50 0)"
    >
      Looking for some food...
    </text>
    <animate
      attributeName="opacity"
      dur="1s"
      keyTimes="0;0.4;1"
      values="0;0;1"
    />
    <g>
      <circle
        cx="60"
        cy="50"
        fill="#22aca7"
        r="4"
      >
        <animate
          attributeName="cx"
          begin="-0.6566000000000001s"
          dur="1.0204081632653061s"
          keyTimes="0;1"
          repeatCount="indefinite"
          values="95;35"
        />
        <animate
          attributeName="fill-opacity"
          begin="-0.6566000000000001s"
          dur="1.0204081632653061s"
          keyTimes="0;0.2;1"
          repeatCount="indefinite"
          values="0;1;1"
        />
      </circle>
      <circle
        cx="60"
        cy="50"
        fill="#22aca7"
        r="4"
      >
        <animate
          attributeName="cx"
          begin="-0.3234s"
          dur="1.0204081632653061s"
          keyTimes="0;1"
          repeatCount="indefinite"
          values="95;35"
        />
        <animate
          attributeName="fill-opacity"
          begin="-0.3234s"
          dur="1.0204081632653061s"
          keyTimes="0;0.2;1"
          repeatCount="indefinite"
          values="0;1;1"
        />
      </circle>
      <circle
        cx="60"
        cy="50"
        fill="#22aca7"
        r="4"
      >
        <animate
          attributeName="cx"
          begin="0s"
          dur="1.0204081632653061s"
          keyTimes="0;1"
          repeatCount="indefinite"
          values="95;35"
        />
        <animate
          attributeName="fill-opacity"
          begin="0s"
          dur="1.0204081632653061s"
          keyTimes="0;0.2;1"
          repeatCount="indefinite"
          values="0;1;1"
        />
      </circle>
    </g>
    <g
      transform="translate(-15 0)"
    >
      <path
        d="M50 50L20 50A30 30 0 0 0 80 50Z"
        fill="#61d8de"
        transform="rotate(90 50 50)"
      />
      <path
        d="M50 50L20 50A30 30 0 0 0 80 50Z"
        fill="#61d8de"
      >
        <animateTransform
          attributeName="transform"
          dur="1.0204081632653061s"
          keyTimes="0;0.5;1"
          repeatCount="indefinite"
          type="rotate"
          values="0 50 50;45 50 50;0 50 50"
        />
      </path>
      <path
        d="M50 50L20 50A30 30 0 0 1 80 50Z"
        fill="#61d8de"
      >
        <animateTransform
          attributeName="transform"
          dur="1.0204081632653061s"
          keyTimes="0;0.5;1"
          repeatCount="indefinite"
          type="rotate"
          values="0 50 50;-45 50 50;0 50 50"
        />
      </path>
    </g>
  </svg>
</div>
`;

exports[`Storyshots Components/TopBanner Default 1`] = `
<div
  className="sc-hKwDye jeEYRB"
>
  <h1
    className="sc-gsDKAQ eMjElI sc-dkPtRN kOpKNy"
  >
    Categories
  </h1>
</div>
`;

exports[`Storyshots Components/Typography/Body Default 1`] = `
<div>
  <p
    className="sc-bdvvtL eMIAGF"
    fontWeight="regular"
    size=""
  >
    Body
  </p>
  <p
    className="sc-bdvvtL eBjqRR"
    fontWeight="regular"
    size="S"
  >
    Body S
  </p>
  <p
    className="sc-bdvvtL iqupKX"
    fontWeight="regular"
    size="XS"
  >
    Body XS
  </p>
  <p
    className="sc-bdvvtL ciRycL"
    fontWeight="regular"
    size="XXS"
  >
    Body XXS
  </p>
</div>
`;

exports[`Storyshots Components/Typography/Heading Default 1`] = `
<div>
  <h1
    className="sc-gsDKAQ eMjElI"
  >
    Heading 1
  </h1>
  <h2
    className="sc-gsDKAQ gXMQwD"
  >
    Heading 2
  </h2>
  <h3
    className="sc-gsDKAQ ecHDGc"
  >
    Heading 3
  </h3>
  <h4
    className="sc-gsDKAQ dWWSwS"
  >
    Heading 4
  </h4>
</div>
`;
